.github/workflows/storybook.yml

```
@@ -1,16 +1,29 @@
 name: Storybook
-on:
-  push:
-    branches: [master]
+on: push
```

```suggestion
on: 
  push:
    paths:
      - 'packages/react-ui/**'
```

---

this will only execute when react-ui has changes

---

Bien vu, merci 🔥

---

Ah génial !

---

Je pensais le mettre de la sorte

```suggestion
on:
  push:
    branches: [master]
````

Pour les runs qu'une fois que c'est merge pour éviter de break certains éléments

---

Je suis d'accord sinon on va casser la preview à chaque PR qui n'est pas forcément acceptée.

---

Pour tester la ci sur cette PR, je laisse en on push là.

Je passerai en master quand ça sera ok